WIP-CHANGELOG.md

```diff
@@ -45,6 +45,7 @@
 - prometheus-operator to v0.63.0
 - grafana to v9.3.8
 - Upgraded Dex to v2.36.0, chart v0.14.1
+- metrics-server to v3.10.0
```

Comment:
please add separately the chart version and the app version

Reply:
Done